### ISSUE TITLE
Can't get path working for demo project

### ISSUE THREAD

**Issue by weavermedia** [jonathonracz/nt_lookandfeel#1]
Hey, I tried to check out the demo project but can't get the path for `nt_lookandfeel` working.

It defaults to `../../` when I open the demo .jucer project but it doesn't see the module.

I tried setting it again manually and also tried absolute paths but nothing worked.

Any ideas?

**Comment by jonathonracz** [comment]
Thanks for the report - I'll look into it later today.

**Comment by jonathonracz** [comment]
Really sorry for taking so long to look into this. When I wrote above I did in fact have intentions of doing so! Unfortunately it fell through the cracks.

I too had the same issue when I downloaded a copy of master, but it was fixed by making sure the folder name of the cloned repository was `nt_lookandfeel`. This is an expectation of Projucer I believe, that a module's folder name will match the name of the module definition file, which in this case is nt_lookandfeel(.h). Let me know if that works for you.

Also, not sure what you're planning on using this for, but keep in mind it's pretty much entirely unfinished - the only notable feature is native font detection for Windows/macOS/iOS, which was a feature I specifically wanted/needed at the time.
